segmentation_models_pytorch/base/model.py

```diff
@@ -27,7 +27,7 @@ def forward(self, x):
         self.check_input_shape(x)
 
         features = self.encoder(x)
-        decoder_output = self.decoder(*features)
+        decoder_output = self.decoder(features)
```

Comment:
This affects all decoders, unet, fpn, etc

Reply:
Yes sorry, I could've made that clearer that I was focusing on just one decoder / encoder pair to get your feedback on the changes. In the future I think it makes sense to add tests to test these conversions for different pairs as well.